addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1061,6 +1061,98 @@ const ColorpickerUserValueWidget = SelectUserValueWidget.extend({
     },
 });
 
+const ImagepickerUserValueWidget = UserValueWidget.extend({
```

Comment:
I have a strange bug with your PR: when I add a background and then hover a background color, the background is removed (as intended) but immediately, the "remove background" button disappears (and he shouldn't as we chose to not show/hide elements during previews). Anyways, the button does not even reappear after the preview and that no color is selected. Could you investigate? Probably a simple bug I guess.

Reply:
Was due to a leftover explicit updateUI in setBgVideo, fixed!